packages/contracts-bedrock/README.md

```diff
@@ -56,11 +56,13 @@ npm install @eth-optimism/contracts-bedrock
 
 We work on this repository with a combination of [Hardhat](https://hardhat.org) and [Foundry](https://getfoundry.sh/).
 
-1. Install Foundry by following [the instructions located here](https://getfoundry.sh/).
-   A specific version must be used.
-   ```shell
-   foundryup -C 3b1129b5bc43ba22a9bcf4e4323c5a9df0023140
-   ```
+1. Install the correct version of foundry (defined in the .foundryrc file in the root of this repo.
+
+```shell
+curl -L https://foundry.paradigm.xyz | bash
+yarn update:foundry
```

Comment:
We should be able to just run a `yarn install:foundry` now right?
```suggestion
yarn install:foundry
```

Reply:
Oh this might also have to be swapped with step 2 since yarn might complain about missing deps